object-construction-checker/stubs/DescribeImages.astub

```diff
@@ -3,13 +3,30 @@ package com.amazonaws.services.ec2;
 import org.checkerframework.checker.objectconstruction.qual.*;
 import org.checkerframework.checker.returnsrcvr.qual.*;
 
+// Interface
 class AmazonEC2 {
     DescribeImagesResult describeImages(
     // any combination of withX/setX has to be permitted if an owner has been set or an imageId has been set or an executable user list has been set
     @CalledMethodsPredicate("(withOwners || setOwners) || (withImageIds || setImageIds) || (withExecutableUsers || setExecutableUsers)")
     DescribeImagesRequest request);
 }
 
+// The main implementation class
+class AmazonEC2Client {
+    DescribeImagesResult describeImages(
+    // any combination of withX/setX has to be permitted if an owner has been set or an imageId has been set
+    @CalledMethodsPredicate("(withOwners || setOwners) || (withImageIds || setImageIds)")
```

Comment:
```suggestion
    @CalledMethodsPredicate("(withOwners || setOwners) || (withImageIds || setImageIds) || (withExecutableUsers || setExecutableUsers)")
```

withExecutableUsers is also sufficient to avoid getting unexpected images.